code/__HELPERS/randoms.dm

```diff
@@ -53,3 +53,15 @@
 	while(length(str) < 5)
 		str = "0" + str
 	. = str
+
+/proc/get_random_seed()
+	var/list/blocked = list(
+		/obj/item/seeds/banana/bombanana,
+		/obj/item/seeds/lavaland,
+		/obj/item/seeds/flower,
+		/obj/item/seeds/sample,
+		/obj/item/seeds/sample/alienweed,
+		/obj/item/seeds/cherry/bomb,
+		/obj/item/seeds/gatfruit
```

Comment:
```suggestion
		/obj/item/seeds/banana/bombanana,
		/obj/item/seeds/lavaland,
		/obj/item/seeds/flower,
		/obj/item/seeds/sample,
		/obj/item/seeds/cherry/bomb
```

I no longer believe gatfruit is too powerful to be an option available here. I'm not sure if the alien weeds seeds will even work, but if they do they should be fine as well and could make for some funny pranks.

This is an optional request, simply resolve it if you still don't want to include them.

Reply:
they don't work, i tested them
but yes I can readd gatfruit